website/blog/2024-10-21-creating-an-extension.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How to create your first extension
+description: Learn how to create and customize your own extension for Podman Desktop
+slug: extensions-getting-started
+authors: [cdrage]
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+---
+
+![programming](img/creating-an-extension/programming.png)
+
+# How to create your first extension
+
+Extensions are a powerful tool to customize and extend the functionality of Podman Desktop. Whether you want to add new container management features, streamline current workflows, or create custom UI elements specific to your tech stack, building extensions allows you to tailor the Podman Desktop experience to your specific needs. 
+
+In this guide, we'll introduce how you can build your own Podman Desktop extension, with links to detailed documentation that covers each part of the process.
+
+## Introduction to Extensions
+
+Extensions are abundant in Podman Desktop and can be found in the **Extensions -> Catalog** section.
+
+![extension catalog](img/podman-desktop-release-1.10/extension-catalog.png)
+
+Each extension expands on Podman Desktop, such as providing [Kubernetes development clusters with Minikube](https://github.com/containers/podman-desktop-extension-minikube) or even [analyzing your images](https://github.com/containers/podman-desktop-extension-layers-explorer).
```

Comment:
```suggestion
Each extension expands on Podman Desktop, such as providing [Kubernetes development clusters with Minikube](https://github.com/containers/podman-desktop-extension-minikube) or even [analyzing your images layers](https://github.com/containers/podman-desktop-extension-layers-explorer).
```

With `analyzing your images` I though about security, when hovering the links I understood you were talking about the layers explorer extension